packages/fx-core/templates/plugins/resource/simpleauth/bicep/simpleAuth.template.bicep

```diff
@@ -0,0 +1,64 @@
+param sku string
+param simpleAuthServerFarmsName string
+param simpleAuthWebAppName string
+param tenantId string
+param AADClientId string
+@secure()
+param AADClientSecret string
+param applicationIdUri string
+{{#contains 'fx-resource-frontend-hosting' plugins}}
+
+param frontendHostingStorageEndpoint string
+{{/contains}}
+
+var oauthAuthorityHost = environment().authentication.loginEndpoint
```

Comment:
it should not be current Azure tenant's endpoint

Reply:
What should it be?

Reply:
The key point is it should not be **Azure**'s login endpoint, it's M365 tenant's login endpoint, which might be different than Azure's. For example, the M365 tenant is a national instance but Azure is a global instance: https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud#azure-ad-authentication-endpoints
We may need to ask AAD plugin to provide this information.

Reply:
Ok. That makes sense. Let me move it to be a input parameter.

Reply:
Done